cc1120_arduino_tests/cc1120_spi.h

```diff
@@ -0,0 +1,203 @@
+#ifndef SPI_CC1120_H
+#define SPI_CC1120_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define R_BIT 1 << 7
+#define BURST_BIT 1 << 6
+
```

Comment:
Define the FIFO size (should be 128 bytes based on the datasheet) so we can compare with the length of transmitted data.